Survey/Survey/Sources/Presentation/Modules/Splash/SplashView.swift

```diff
@@ -9,10 +9,26 @@ import SwiftUI
 
 struct SplashView: View {
 
+    @State private var logoOpacity = 0.0
+
     var body: some View {
         ZStack {
-            Color(.systemTeal)
-            Text("Survey App")
+            Image(Images.splashBackground)
+                .resizable()
+                .aspectRatio(contentMode: .fill)
+                .edgesIgnoringSafeArea(.top)
```

Comment:
I wonder why we ignore only the `.top` edge here 🤔 Should it ignore the other edges as well, or at least `.top` and `.bottom`?